### PR TITLE
Feature #1

### PR DESCRIPTION
**Pull request by denis**
The user can launch an editing pane in a browser by visiting http://barushev.net/shaper/builds/1/

What's done:
- Created project skeleton
- Learned how to use NPM's package.json
- Configured TypeScript, Typings and webpack
- Created a React component
- Started learning JSX

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
+        "outDir": "./dist/",
+        "sourceMap": true,
+        "noImplicitAny": true,
+        "module": "commonjs",
+        "target": "es5",
+        "jsx": "react"
+    },
+    "files": [
```
Comment (danschultz) [inline tsconfig.json]: i dont believe `files` is necessary since you're running builds through webpack.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "postinstall": "./node_modules/.bin/typings install"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "react": "^15.2.1",
+    "react-dom": "^15.2.1",
+    "typescript": "^1.8.10",
+    "typings": "^1.3.1",
+    "webpack": "^1.13.1"
```
Comment (danschultz) [inline package.json]: `typescript`, `typings`, and `webpack` should be dev dependencies.

Comment (danschultz) [inline package.json]: here's some info on normal, dev and peer dependencies if you'd like to read more: https://github.com/npm/npm/blob/2e3776bf5676bc24fec6239a3420f377fe98acde/doc/files/package.json.md#devdependencies

**Comment by danschultz** [comment]
LGTM, just some minor stuff

**Comment by denis** [comment]
@danschultz thanks for the feedback.

**Pull request merged**